Create Header and include in App.render

sanfordcb opened the pull request.

sanfordcb committed:
Create Header and include in App.render

Thr1ve commented:
It looks like you've committed old files to this PR. Could we remove them from the PR or perhaps open a new PR with just the files you're adding?

sanfordcb committed:
Cleanup, Remove public folder

sanfordcb commented:
I removed the public folder from my branch.  I'm going to close this pull request and submit a new one.

sanfordcb commented:
I'm sorry about that.  I removed them, closed the pull request, and
submitted a new one with the updated branch.  I have a commit showing that
I deleted the public folder, so I hope that isn't an issue.

On Wed, Feb 24, 2016 at 3:55 PM, Gordon Buhler notifications@github.com
wrote:

> It looks like you've committed old files to this PR. Could we remove them
> from the PR or perhaps open a new PR with just the files you're adding?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/HypnoSheep/HypnoSheep/pull/70#issuecomment-188469343.
